_pkgdown.yml

```diff
@@ -21,27 +21,29 @@ reference:
 
 - title: Individual linters
   contents:
-  - all_undesirable_functions
   - ends_with("linter")
 
-- title: Customer linters
+- title: Lists of defaults
+  contents:
+  - all_undesirable_functions
+  - starts_with("default_")
```

Comment:
So that all named lists are together in one section:

<img width="356" alt="Screenshot 2022-07-29 at 00 31 15" src="https://user-images.githubusercontent.com/11330453/181648132-8398236f-dbf9-4a65-b4f9-b62e3bf94476.png">